### DISCUSSION TITLE
Pico W usb serial issue

### DISCUSSION THREAD

**Discussion by buhlerles** [earlephilhower/arduino-pico#1627] [Q&A] [answered]
Working with Arduino and Earle's amazing library but have a very strange issue when connecting to the builtin USB serial port.
Connections with the Arduino serial monitor work perfectly but any attempt to use a VB.net program with built-in windows serial objects does not recognize data transmitted FROM the pico?? This same program works perfectly with other processors such as the Teensy.
I have independently verified that the pico is receiving data sent TO it.
Strangly enough, most Terminal programs work, but some exhibit the same behavior as above.
Any ideas would be greatly appreciated!

**Comment by earlephilhower** [comment, marked as answer]
It's very odd to come up right now, but there was just an addition to ignore the virtual DTR bit when writing to the USB serial port.  I've never see anything that didn't set it properly from the PC, but evidently it's possible if you try hard enough. :)

See the docs:  https://github.com/earlephilhower/arduino-pico/blob/master/docs/serial.rst#rp2040-specific-serialusb-methods

You'll need to use the `git` head as this was literally added less than 24 hours ago, but by adding a `Serial.ignoreFlowControl(true)` to the top of your `setup()` it might just be all you need.

**Reply by buhlerles** [reply]
You are a lifesaver! This would certainly explain the issue.
I'll need to wait for v3.3.2 because unfortunately I'm not proficient enough on GitHub to figure out how to do the update before a release, but thank-you again for solving such a niche problem!

**Reply by buhlerles** [reply]
v3.2.2 fixed the issue - thanks again!

**Reply by earlephilhower** [reply]
Great, appreciate the follow up!
